index.bs

```diff
@@ -41,6 +41,9 @@ spec: mediacapture-main; urlPrefix: https://w3c.github.io/mediacapture-main/#
 spec: sensors; urlPrefix: https://w3c.github.io/sensors/#
     type: dfn
         text: generic sensor permission revocation algorithm; url: generic-sensor-permission-revocation-algorithm
+spec: manifest; urlPrefix: https://w3c.github.io/manifest/#
+    type: dfn
+        text: install; url: dfn-install
```

Comment:
Once https://github.com/tabatkins/bikeshed/issues/1088 is fixed, you should be able to remove this.

Reply:
I will leave this in while waiting for that upstream patch to land, ok?

Reply:
Yep.